### ISSUE TITLE
it seems that we no longer have continuous integration tests (AppVeyor and Travis).

### ISSUE THREAD

**Issue by servoz** [populse/populse_mia#156]
Not yet investigated ...

**Comment by denisri** [comment]
You're right, I only see my personal forks on travis-ci.org. Strange. Maybe it's time to move to travis-ci.com which is their newer server.

**Comment by denisri** [comment]
Well, they seem to still exist and run (https://travis-ci.org/github/populse/populse_mia/builds/666737372?utm_medium=notification&utm_source=github_status) but I don't see them any longer in the list on the left panel...

**Comment by denisri** [comment]
And in travis-ci.com I can't see all populse repositories.
All this is mysterious to me...

**Comment by denisri** [comment]
things have come back on travis-ci. It must have been something weird on their side.

**Comment by servoz** [comment]
Yes, you're right. It does seem to be coming back for Travis. But not for AppVeyor (if we look at the result on the [commits page](https://github.com/populse/populse_mia/commits/master)). Also surprising, the result for Travis AND AppVeyor (not only Travis) is now available on your last PR .... Really mysterious story !

**Comment by denisri** [comment]
Maybe the problem was on github...?

**Comment by denisri** [comment]
There seems to be a random segfault in the tests, in travis-ci (I haven't seen in appveyor). restarting the test can make it pass. However I agree this is not a good long term situation...

**Comment by servoz** [comment]
Yes, we've random segmentation errors or test failures for some time... I try to restart by hand when I see failures in the commits page. I never took the time to understand it, maybe I'll have to do it (I don't know if the problem comes from us or from the appveyor / travis / github ... that's also why I'm a bit afraid to look at it ... for nothing ...).

Anyway, at the moment :
- In the [mia project page](https://github.com/populse/populse_mia):
  - if we click the badges for travis and appveyer, we are redirected to the tests of the last commits (currently the PR merge). Ok ! 

- on the [commits page](https://github.com/populse/populse_mia/commits/master):
  - if we click on the check of commit "fix unitary test check" of march 7th (e2e7f94), we see 4 checks reported:
    - Page Build — GitHub Pages successfully built your site.
    - continuous-integration/appveyor/branch — AppVeyor build succeeded
    - continuous-integration/travis-ci/push — The Travis CI build passed
    - github/pages — GitHub Pages successfully built your site.

  - if we click on the last commit (777208c), only 3 checks are reported :
    - Page Build - GitHub Pages has successfully built your site.
    - Continuous Integration/travis-ci/push - IC Travis build was successful
    - github/pages - GitHub Pages has successfully built your site.

In summary, on the [mia project page](https://github.com/populse/populse_mia), the badges giving the status of the tests seems ok, but we lost this status for all the commits after March 7th regarding AppVeyor build in the [commits page](https://github.com/populse/populse_mia/commits/master) ...
I've looked a little bit without finding any particular problem. If someone wants to look, maybe I'm forgetting something?

**Comment by servoz** [comment]
This issue seems now fixed. [The migration to GitHub Apps for the existing AppVeyor account is now effective](https://help.appveyor.com/discussions/problems/26712-github-account-change-for-appveyors-authorization-to-act). GitHub App acts on its own behalf and not as a GitHub user authorizing the App, as it was the case before. I close this ticket.
